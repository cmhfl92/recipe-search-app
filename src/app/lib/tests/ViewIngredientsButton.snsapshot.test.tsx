import { render, screen, fireEvent } from '@testing-library/react';
import React from 'react';

const ViewIngredientsButtonTest = ({
  _id,
  ingredients,
  label,
}: {
  _id: string;
  ingredients: string[];
  label: string;
}) => {
  const clickHandler = (_id: string) => {};

  return <button onClick={() => clickHandler(_id)}>View Ingredients</button>;
};

describe('View Ingredients Button', () => {
  it('views ingredients via modal', () => {
    const { asFragment } = render(
      <ViewIngredientsButtonTest
        _id='test_recipe_1'
        label='test_ingredients_1'
        ingredients={['ingredient test 1', 'ingredient test 2']}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
