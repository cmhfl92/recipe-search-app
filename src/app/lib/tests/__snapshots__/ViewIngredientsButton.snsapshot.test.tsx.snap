// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Ingredients Button views ingredients via modal 1`] = `
<DocumentFragment>
  <button>
    View Ingredients
  </button>
</DocumentFragment>
`;
